00XX-hst-realignment.md

```diff
@@ -0,0 +1,116 @@
+# HIP ##: HST Realignment
+
+- Start Date: 2023/05/11
+- Original HIP PR: [#XX](https://github.com/helium/HIP/pull/XX)
+- Tracking Issue: [#XX](https://github.com/helium/HIP/issues/XX)
+- Authors: [@seidenbergerscott](https://github.com/seidenbergerscott), [@sokolalec](https://github.com/sokolalec)
+
+# Summary
+
+_"Aligning HST Distribution for the next wave of Helium adoption."_
+
+This HIP addresses the misalignment in the distribution of HNT emissions skewed by HST. This HIP 
+proposes that HST emissions be reduced to 5% of HNT emissions. In doing so, a larger and more
+inclusive share of HNT will be emitted to current and future subDAOs, the primary engine of 
+the Helium Network of Networks. 
+
+# Motivation
+
+The initial token distribution was decided when Helium first began in 2013. Ten years later,
+Helium and its community has grown and evolved to be a highly functional, self-governing community.
+Early ventures require some form of capital incentive in order to attract talented core developers
+who can turn a vision into reality. The Helium community at-large is grateful and committed to the
+core development teams that have propelled the People's Network to becoming the world's largest,
+permissionless, decentralized network of networks. However, the HST token distribution which rewarded early builders and investors currently hinders its continued growth. 
+
+The Helium network is now separated from the original corporate entity, Helium Inc., and is now run
+by the Helium Foundation, which is charged with maintaining an equitable Helium ecosystem for all
```

Comment:
This isn't a true statement and should be removed.
The Helium Foundation stewards the network. They take all direction from the Community and community governance. So they don't really run the network, more so implement what the community wants and decides. 

You could say something like:
" The Helium Network is now separated from the original corporate entity, Helium Inc., and is now decentralized to the community on decision making and stewarded by the Helium Foundation..."